### PR TITLE
Remove CircleCI badge from README.md #trivial

### PR DESCRIPTION
**Pull request by martinpucik**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Removes no longer used CircleCI status bagge

Does this close any currently open issues?
------------------------------------------
No

Any relevant logs, error output, etc?
-------------------------------------

Any other comments?
-------------------
N/A

### CODE REVIEW

**Comment by martinpucik** [comment]
Badges with status from Github Actions
Screenshot 2020-06-02 at 18 48 54

**Comment by kinoroy** [comment]
I think right now we only run CI for PRs so would this only show the status of the last PR merged (whether or not that went to master or dev or some other branch)? Should we run build/test also on commits to master or dev so we can display the badge for those branches instead?

**Comment by martinpucik** [comment]
@kinoroy I think there is an option to filter the badge for a specific branch. I’ll check. 
That should be enough tho and we wouldn’t need another set of actions to run on master pushes

**Comment by kinoroy** [comment]
You're right, actually it says [here](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository) that the badges show the state of the master branch by default. So I guess its just showing the latest ran PR that merged to the master branch which is probably good enough since we should all be using PRs to commit to master.

**Review by kinoroy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,7 +4,12 @@
   A community-driven replacement for JSQMessagesViewController https://messagekit.github.io
 </p>
 
-[![CircleCI](https://circleci.com/gh/MessageKit/MessageKit.svg?style=svg)](https://circleci.com/gh/MessageKit/MessageKit)
+[![tests](https://github.com/MessageKit/MessageKit/workflows/PR%20Tests/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Tests%22)
+[![framework](https://github.com/MessageKit/MessageKit/workflows/PR%20Framework/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Framework%22)
+[![example](https://github.com/MessageKit/MessageKit/workflows/PR%20Example%20app/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Example+app%22)
+[![danger](https://github.com/MessageKit/MessageKit/workflows/PR%20Danger/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Danger%22)
```
Comment (mh-austin) [inline README.md:10]:
```suggestion
[![Tests](https://github.com/MessageKit/MessageKit/workflows/PR%20Tests/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Tests%22)
[![Framework](https://github.com/MessageKit/MessageKit/workflows/PR%20Framework/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Framework%22)
[![Example](https://github.com/MessageKit/MessageKit/workflows/PR%20Example%20app/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Example+app%22)
[![Danger](https://github.com/MessageKit/MessageKit/workflows/PR%20Danger/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Danger%22)
```

Comment (austinwright) [inline README.md]:
```suggestion
[![Tests](https://github.com/MessageKit/MessageKit/workflows/PR%20Tests/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Tests%22)
[![Framework](https://github.com/MessageKit/MessageKit/workflows/PR%20Framework/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Framework%22)
[![Example](https://github.com/MessageKit/MessageKit/workflows/PR%20Example%20app/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Example+app%22)
[![Danger](https://github.com/MessageKit/MessageKit/workflows/PR%20Danger/badge.svg)](https://github.com/MessageKit/MessageKit/actions?query=workflow%3A%22PR+Danger%22)
```

**Pull request merged**